main/acle.md

```diff
@@ -9382,6 +9415,2316 @@ are named after. All of the functions have external linkage.
+  __attribute__((arm_streaming))
+  svcount_t svwhilelt_c8(int64_t rn, int64_t rm, uint64_t vl);
+  
+  ```
+
+#### WHILEGE, WHILEGT, WHILEHI, WHILEHS, WHILELE, WHILELO, WHILELS, WHILELT
+
+While (resulting in predicate tuple)
+
+``` c
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilege_b8_x2(int64_t rn, int64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilegt_b8_x2(int64_t rn, int64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilehi_b8_x2(uint64_t rn, uint64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilehs_b8_x2(uint64_t rn, uint64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilele_b8_x2(int64_t rn, int64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilelo_b8_x2(uint64_t rn, uint64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilels_b8_x2(uint64_t rn, uint64_t rm);
+  
+  
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svboolx2_t svwhilelt_b8_x2(int64_t rn, int64_t rm);
+  
+  ```
+
+#### SUNPK, UUNPK
+
+Multi-vector pack/unpack
+
+``` c
+  // Variants are also available for _u16_x2, _u32_x2, _s32_x2, _u64_x2 and _s64_x2
+  __attribute__((arm_streaming))
+  svint16x2_t svunpk[_s16_x2](svint8_t zn);
+  
+  
+  // Variants are also available for _u16_x4, _u32_x4, _s32_x4, _u64_x4 and _s64_x4
+  __attribute__((arm_streaming))
+  svint16x4_t svunpk[_s16_x4](svint8x2_t zn);
+  
+  ```
+
+#### ZIP, UZP
+
+Multi-vector zip/unzip (2 vectors)
+
+The uzipq instructions operate on quad-words, but for convenience accept all element types.
+
+``` c
+  // Variants are also available for _u8, _u16, _s16, _f16, _bf16, _u32, _s32, _f32,
+  // _u64, _s64 and _f64
+  __attribute__((arm_streaming))
+  svint8x2_t svuzp[_s8]_x2(svint8_t zn, svint8_t zm);
+  
+  
+  // Variants are also available for _u8, _u16, _s16, _f16, _bf16, _u32, _s32, _f32,
+  // _u64, _s64 and _f64
+  __attribute__((arm_streaming))
+  svint8x2_t svuzpq[_s8]_x2(svint8_t zn, svint8_t zm);
+  
+  
+  // Variants are also available for _u8, _u16, _s16, _f16, _bf16, _u32, _s32, _f32,
+  // _u64, _s64 and _f64
+  __attribute__((arm_streaming))
+  svint8x2_t svzip[_s8]_x2(svint8_t zn, svint8_t zm);
+  
+  
+  // Variants are also available for _u8, _u16, _s16, _f16, _bf16, _u32, _s32, _f32,
+  // _u64, _s64 and _f64
+  __attribute__((arm_streaming))
+  svint8x2_t svzipq[_s8]_x2(svint8_t zn, svint8_t zm);
+  
+  ```
+
+#### ZIP, UZP
```

Comment:
Not sure it's worth spitting the `x2` and `x4` functions into separate blocks.  It's probably more convenient to the reader if a `x4` function comes immediately after any corresponding `x2` function, which seems to be the style followed in the other sections.